cmd/ddev/cmd/composer-create.go

```diff
@@ -79,20 +78,20 @@ ddev composer create --prefer-dist --no-interaction --no-dev psr/log
 
 		// Remove most contents of composer root
 		util.Warning("Removing any existing files in composer root")
-		objs, err := fileutil.ListFilesInDir(composerRoot)
-		if err != nil {
-			util.Failed("Failed to create project: %v", err)
-		}
 
-		for _, o := range objs {
-			// Preserve .ddev, .git, .tarballs
-			if o == ".ddev" || o == ".git" || o == ".tarballs" {
-				continue
+		skip := []string{}
+		if app.IsMutagenEnabled() {
```

Comment:
I think we should do this whether or not mutagen is enabled? No reason to remove those if they're upload_dirs. I know the issue is about when mutagen is enabled, but I don't think we should be removing the upload_dirs normally either.